test/test_add_data.py

```diff
@@ -28,14 +28,13 @@ def test_add_all_data():
 
 
 def test_add_node_data():
-
     grid = two_triangles()
 
     # Create a UVar object for the depths:
     depths = UVar('depth', location='node', data=[1.0, 2.0, 3.0, 4.0])
     depths.attributes['units'] = 'm'
-    depths.attributes["standard_name"] = "sea_floor_depth"
-    depths.attributes["positive"] = "down"
+    depths.attributes['standard_name'] = 'sea_floor_depth'
```

Comment:
Is there some standard I don't know about for when to use single vs double quotes?

PEP8 says:
"""
In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it.
"""

so what's the rule for this project?